### DISCUSSION TITLE
关于 Netlify 部署后的疑问 || Questions about Netlify after deployment

### DISCUSSION THREAD

**Discussion by nibazshab** [walinejs/waline#2520] [General]
如果我就准备拿服务端当作客户端使用，也就是那个 `xxxx.netlify.app/.netlify/functions/comment/`，怎么才能做到直接访问 `xxxx.netlify.app` 就好，而不用加 `.netlify/functions/comment/` 这一长串，要怎么做呢？

---
If I plan to use the server as a client, that is, `xxxx.netlify.app/.netlify/functions/comment/`, how can I directly access `xxxx.netlify.app` without using How to add `.netlify/functions/comment/` to this long list?

**Comment by lizheming** [comment]
这个没办法，netlify 的 serverless URL 格式就是这样的，要不然你就自己域名跳转到这个地址。

---
There's no way around this. Netlify's serverless URL format is like this, otherwise you can just jump to this address with your own domain name.
